integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -10,6 +10,9 @@ set -e
 source /etc/os-release || source /usr/lib/os-release
 kubernetes_dir=$(dirname $0)
 
+# If NODES is not already defined, default it to 1.
+NODES="${NODES:-1}"
```

Comment:
Are we setting NODES somewhere else as well them, such as in the Jenkins job ENV section?